Change the link to the page that lists time zone names

davidebbo opened the pull request:
Original page https://msdn.microsoft.com/library/ms912391.aspx uses zone names that in some cases do not work, whereas https://technet.microsoft.com/en-us/library/cc749073(v=ws.10).aspx has the correct names. e.g. "A.U.S. Central Standard Time" is wrong, vs "AUS Central Standard Time" is correct.

azurecla commented:
Hi __@minghan__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

davidebbo committed:
Change the link to the page that lists time zone names

Original page https://msdn.microsoft.com/library/ms912391.aspx uses zone names that in some cases do not work, whereas https://technet.microsoft.com/en-us/library/cc749073(v=ws.10).aspx has the correct names. e.g. "A.U.S. Central Standard Time" is wrong, vs "AUS Central Standard Time" is correct.

tynevi committed:
Update functions-bindings-timer.md
